.github/workflows/main.yaml

```diff
@@ -7,6 +7,8 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
+      - name: Upgrade Jinja
+        run: pip install --upgrade jinja2
```

Comment:
Can't this be part of `requirements.txt`?

Reply:
Fixed